tally_ho/urls.py

```diff
@@ -61,10 +61,10 @@
          name='logout'),
 ]
 
-handler403 = 'tally_ho.apps.tally.views.home.permission_denied'
-handler404 = 'tally_ho.apps.tally.views.home.not_found'
-handler400 = 'tally_ho.apps.tally.views.home.bad_request'
-handler500 = 'tally_ho.apps.tally.views.home.server_error'
+# handler403 = 'tally_ho.apps.tally.views.home.permission_denied'
+# handler404 = 'tally_ho.apps.tally.views.home.not_found'
+# handler400 = 'tally_ho.apps.tally.views.home.bad_request'
+# handler500 = 'tally_ho.apps.tally.views.home.server_error'
```

Comment:
@JohnMwashuma Should this be removed, It seems like its dead code, but I am not sure.